src/main/kotlin/org/rust/ide/refactoring/convertStruct/RsConvertToTupleProcessor.kt

```diff
@@ -0,0 +1,128 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.refactoring.convertStruct
+
+import com.intellij.openapi.progress.ProgressManager
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiReference
+import com.intellij.psi.search.searches.ReferencesSearch
+import com.intellij.refactoring.BaseRefactoringProcessor
+import com.intellij.usageView.BaseUsageViewDescriptor
+import com.intellij.usageView.UsageInfo
+import com.intellij.usageView.UsageViewDescriptor
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.ext.*
+
+class RsConvertToTupleProcessor(project: Project, val element: RsFieldsOwner, val convertUsages: Boolean = true) : BaseRefactoringProcessor(project) {
+    private val rsPsiFactory = RsPsiFactory(project)
+    private val fieldDeclList = element.blockFields!!.namedFieldDeclList
+    override fun findUsages(): Array<UsageInfo> {
+        if (!convertUsages) return arrayOf()
+
+        var usages = ReferencesSearch
+            .search(element, element.useScope)
```

Comment:
As far as I know, `element.useScope` isn't reasonable here b/c `ReferencesSearch` internally retrieves `element.useScope` end intersects in with an argument scope